aggregate.go

```diff
@@ -78,19 +84,32 @@ type spanGroup struct {
 	Slowest []spanGroupSlowest // N-slowest traces in the group.
 }
 
+func (s spanGroup) Len() int      { return len(s.Slowest) }
+func (s spanGroup) Swap(i, j int) { s.Slowest[i], s.Slowest[j] = s.Slowest[j], s.Slowest[i] }
+func (s spanGroup) Less(i, j int) bool {
+	a := s.Slowest[i]
+	b := s.Slowest[j]
+	return a.End.Sub(a.Start) > b.End.Sub(b.Start)
```

Comment:
It's Intentionally that way: the sorting places the N-slowest traces in order of slowest-to-fastest (i.e. ones that took _most time_ to ones that took _least time_), so it can be read as "A sorts before B if it took a greater amount of time than B".